src/Sylius/Behat/Page/Admin/Promotion/UpdatePage.php

```diff
@@ -92,6 +92,29 @@ public function hasEndsAt(\DateTimeInterface $dateTime): bool
             && $this->getElement('ends_at_time')->getValue() === date('H:i', $timestamp);
     }
 
+    public function hasAnyRule(): bool
+    {
+        $items = $this->getElement('rules')->findAll('css', 'div[data-form-collection="item"]');
+
+        return 0 > count($items);
```

Comment:
Shouldn't it be like this:
```suggestion
        return 0 < count($items);
```
?